+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n '' ']'
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Sat Dec  3 12:55:42 2022 - INFO  - Start to run test.
+ echo 'Hello, world!'
+ openssl passwd -crypt -salt 12345 root
oe_test_openssl_generate_password.sh: line 24: openssl: command not found
+ CHECK_RESULT 127
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_openssl_generate_password.sh line 25'
+ message='oe_test_openssl_generate_password.sh line 25'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_openssl_generate_password.sh line 25'
Sat Dec  3 12:55:43 2022 - ERROR - oe_test_openssl_generate_password.sh line 25
+ return 0
+ openssl passwd -crypt -in word -salt 12345
oe_test_openssl_generate_password.sh: line 26: openssl: command not found
+ CHECK_RESULT 127
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_openssl_generate_password.sh line 27'
+ message='oe_test_openssl_generate_password.sh line 27'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_openssl_generate_password.sh line 27'
Sat Dec  3 12:55:44 2022 - ERROR - oe_test_openssl_generate_password.sh line 27
+ return 0
+ openssl passwd -1 -in word -salt 12345
oe_test_openssl_generate_password.sh: line 28: openssl: command not found
+ CHECK_RESULT 127
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_openssl_generate_password.sh line 29'
+ message='oe_test_openssl_generate_password.sh line 29'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_openssl_generate_password.sh line 29'
Sat Dec  3 12:55:45 2022 - ERROR - oe_test_openssl_generate_password.sh line 29
+ return 0
+ expect
spawn openssl passwd -1 -salt 12345 -stdin
couldn't execute "openssl": no such file or directory
    while executing
"spawn openssl passwd -1 -salt 12345 -stdin"
+ grep 111 testlog1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_openssl_generate_password.sh line 43'
+ message='oe_test_openssl_generate_password.sh line 43'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_openssl_generate_password.sh line 43'
Sat Dec  3 12:55:46 2022 - ERROR - oe_test_openssl_generate_password.sh line 43
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Sat Dec  3 12:55:47 2022 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 4
+ test 4 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Sat Dec  3 12:55:47 2022 - ERROR - failed to execute the case.
+ exit 4
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Sat Dec  3 12:55:48 2022 - INFO  - Start to restore the test environment.
+ rm -f word testlog1
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Sat Dec  3 12:55:49 2022 - INFO  - End to restore the test environment.
